lib/Elastica/Multi/Search.php

```diff
@@ -27,18 +32,20 @@ class Search
     protected $_options = array();
 
     /**
-     * @var \Elastica\Client
+     * @var array|\Elastica\Search[]
      */
-    protected $_client;
+    protected $_searches = array();
 
     /**
      * Constructs search object.
      *
      * @param \Elastica\Client $client Client object
+     * @param MultiBuilderInterface $builder
      */
-    public function __construct(Client $client)
+    public function __construct(Client $client, MultiBuilderInterface $builder = null)
     {
-        $this->setClient($client);
+        $this->_builder = $builder ?: new MultiBuilder();
```

Comment:
In a way there are 2 default builders: One for normal requests, one for multi requests. In the client only the one for single requests can be set?